spec/models/comfy/cms/page_spec.rb

```diff
@@ -1,5 +1,57 @@
 RSpec.describe Comfy::Cms::Page do
 
+  describe 'suppress_from_links_recirculation' do
+    let(:english_site) { create :site, is_mirrored: true }
+    let(:welsh_site) { create :site, :welsh, is_mirrored: true }
+
+    let!(:english_article) do
+      create :english_article,
+             full_path:  '/animals',
+             site: english_site
+    end
+    let!(:welsh_article) do
+      create :welsh_article,
+             full_path:  '/animals',
+             site: welsh_site
+    end
+
+    context 'update welsh article when updating english_article' do
+      it 'to be suppressed' do
+        english_article.suppress_from_links_recirculation = true
+        english_article.suppress_mirrors_from_links_recirculation
+        welsh_article.reload
+
+        expect(welsh_article.suppress_from_links_recirculation).to be_truthy
+      end
+
+      it 'to not be suppressed' do
+        english_article.suppress_from_links_recirculation = false
+        english_article.suppress_mirrors_from_links_recirculation
+        welsh_article.reload
+
+        expect(welsh_article.suppress_from_links_recirculation).to be_falsey
+      end
+    end
+
+    context 'update english article when updating welsh article' do
+      it 'to be suppressed' do
+        welsh_article.suppress_from_links_recirculation = true
+        welsh_article.suppress_mirrors_from_links_recirculation
+        english_article.reload
+
+        expect(english_article.suppress_from_links_recirculation).to be_truthy
+      end
+
+      it 'to not be suppressed' do
+        welsh_article.suppress_from_links_recirculation = false
+        welsh_article.suppress_mirrors_from_links_recirculation
+        english_article.reload
+
```

Comment:
Yes, exactly, the boolean is different and the expectation too. Given that:

```ruby
context 'update english article when updating welsh article' do
  before do
    welsh_article.suppress_from_links_recirculation = supressed?
    welsh_article.suppress_mirrors_from_links_recirculation
    english_article.reload
  end
  
  context 'when welsh article supressed' do
    let(:supressed?) { true }
  
    it 'english article should be supressed' do
      expect(english_article.suppress_from_links_recirculation).to be_truthy
    end
  end
  
  context 'when welsh article not supressed' do
    let(:supressed?) { false }
  
    it 'english article should not be supressed' do
      expect(english_article.suppress_from_links_recirculation).to be_falsey
    end
  end
end
```

Something like that (I didn't tested, is just a draft).